app/views/admin/events/index.html.erb

```diff
@@ -18,19 +18,18 @@
 
   <tbody>
     <% @events.each do |event| %>
+    <% locationType = JSON.parse(event.location) %>
       <tr>
         <td><%= event.name %></td>
         <td><%= event.description %></td>
         <td><%= image_tag event.image.url(:thumb) %></td>
         <td><%= event.url %></td>
         <td><%= event.date %></td>
-        <td><%= event.location %></td>
+        <td><%= locationType["geometry"]["type"] %></td>
```

Comment:
if there's no location_type or if location_type's hash doesn't have a 'geometry' this will break.

worth using here:

`location_type["geometry"] && location_type["geometry"]["type"] if location_type`

(if there's a location_type, and a geomtry, check the type)

After writing this I realised that we probably should change the data type of "event.location" from string to json. You can change it by creating a migration to change the column type. Useful links:

* http://edgeguides.rubyonrails.org/active_record_migrations.html
* http://edgeguides.rubyonrails.org/active_record_postgresql.html#json
* http://apidock.com/rails/ActiveRecord/ConnectionAdapters/SchemaStatements/change_column